sdk/eventhubs/azure-messaging-eventhubs/src/test/java/com/azure/messaging/eventhubs/implementation/EventHubReactorConnectionTest.java

```diff
@@ -158,6 +160,13 @@ public void getsManagementChannel() {
         StepVerifier.create(connection.getManagementNode())
             .assertNext(node -> Assertions.assertTrue(node instanceof ManagementChannel))
             .verifyComplete();
+
+        StepVerifier.create(connection.createSendLink("send-name", "test-entity-path", new AmqpRetryOptions()))
+            .verifyError();
+
+        StepVerifier.create(connection.createReceiveLink("receiver-name", "test-entity-path", EventPosition.latest(), new ReceiveOptions()))
+            .verifyError();
```

Comment:
What's the error we are verifying here? Can we also assert on the expected error?